Source/Core/Common/StringUtil.cpp

```diff
@@ -77,13 +77,10 @@ bool AsciiToHex(const std::string& _szValue, u32& result)
   // Set errno to a good state.
   errno = 0;
 
-  char* endptr = nullptr;
-  const u32 value = strtoul(_szValue.c_str(), &endptr, 16);
+  const u32 value = strtoul(_szValue.c_str(), nullptr, 16);
 
-  if (!endptr || *endptr)
```

Comment:
According to the  `Open STD page 345`

If the endptr is NULL, then nothing is stored there. So is this check for future purposes; if the nullptr changes in the future to something else?